apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsTextControl.tsx

```diff
@@ -72,7 +72,7 @@ export function JsonFormsTextControl({
           onChange={onChange}
           placeholder={label}
           maxLength={maxLength}
-          my="0.5rem"
+          mt="0.5rem"
```

Comment:
issue: I don't think we can do this hmm, it's causing the helper text to be too close to the input field
<img width="719" height="248" alt="image" src="https://github.com/user-attachments/assets/1bd7c0d3-7c81-4d1c-a52c-99860640d6eb" />

Reply:
reverted